GPU/GLES/StateMapping.cpp

```diff
@@ -384,11 +388,14 @@ void TransformDrawEngine::ApplyBlendState() {
 		default:
 			break;
 		}
+		needsConstantAlpha = constantAlpha < 1.0f;
```

Comment:
Well, currently it's not smart enough to optimize out 0, but yeah, that should be doable...

-[Unknown]